### PR TITLE
Update TokenVesting.sol

### PR DESCRIPTION
**Pull request by developer-asiatop**
upgrade to 8.4

### CODE REVIEW

File: contracts/TokenVesting.sol [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.3;
+pragma solidity ^0.8.4;
```
Comment (developer-asiatop) [inline contracts/TokenVesting.sol:2]: good change

**Review by developer-asiatop — COMMENTED** [review]
good. why not

**Comment by developer-asiatop** [comment]
review completed

**Comment by developer-asiatop** [comment]
please close

**Review by dragonbite-dev — COMMENTED** [review]
goood

**Comment by dragonbite-dev** [comment]
good

**Pull request merged**